packages/zudoku/src/lib/plugins/openapi/ParameterListItem.tsx

```diff
@@ -15,6 +17,32 @@ const getParameterSchema = (
   };
 };
 
+const EnumValues = ({ values }: { values: any[] }) => {
+  const [showEnums, setShowEnums] = useState(false);
+
+  if (values.length <= 5 || showAllEnums) {
```

Comment:
```suggestion
  if (values.length <= 5 || showEnums) {
```

showAllEnums -> showEnums